text/0000-trait-alias.md

```diff
@@ -150,6 +150,22 @@ more predicates, just like any other `where` clause.
 You cannot directly `impl` a trait alias, but you can have them as *bounds*, *trait objects* and
 *impl Trait*.
 
+----
+
+It is an error to attempt to override a previously specified
+equivalence constraint with a non-equivalent type. For example:
+
+```rust
+trait SharableIterator = Iterator + Sync;
+trait IntIterator = Iterator<Item=i32>;
+
+fn quux1<T: SharableIterator<Item=f64>>(...) { ... } // ok
+fn quux2<T: IntIterator<Item=i32>>(...) { ... } // ok (perhaps subject to lint warning)
```

Comment:
Well, I think you’re right about the consistency. Others construct follow such a pattern. I’ll merge it then.